presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplitManager.java

```diff
@@ -100,7 +99,8 @@ public ConnectorSplitSource getSplits(ConnectorTransactionHandle transaction, Co
                 SimpleConsumer leaderConsumer = consumerManager.getConsumer(partitionLeader);
                 // Kafka contains a reverse list of "end - start" pairs for the splits
 
-                List<HostAddress> partitionNodes = ImmutableList.copyOf(Lists.transform(part.isr(), KafkaSplitManager::brokerToHostAddress));
+                // Always talk to partition leader
+                List<HostAddress> partitionNodes = ImmutableList.of(partitionLeader);
```

Comment:
A list of nodes here can be eliminated, but maybe it's better to keep the list of nodes in ```KafkaSplit```, because ```KafkaSplit``` should override function ```getAddresses()``` for 
scheduling, and this returns a list of HostAddress.
```split.getNodes().get(0)```can be simplified as ```split.getLeader()```.

What do you suggest? Should we build a list everytime we call ```getAddresses()```.